src/github/schemas.ts

```diff
@@ -285,94 +289,367 @@ export const GitHubPullRequestSchema = z.object({
+      patch_url: z.string().describe("The URL to view the patch"),
+    })
+    .optional()
+    .describe("Pull request details if this is a PR"),
+});
+
+/**
+ * Response schema for issue search results
+ */
+export const SearchIssuesResponseSchema = z.object({
+  total_count: z.number().describe("Total number of matching results"),
+  incomplete_results: z
+    .boolean()
+    .describe("Whether the results are incomplete"),
+  items: z.array(SearchIssueItemSchema).describe("The search results"),
+});
+
+/**
+ * Response schema for a user search result item
+ * @see https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-users
+ */
+export const SearchUserItemSchema = z.object({
+  login: z.string().describe("The username of the user"),
+  id: z.number().describe("The ID of the user"),
+  node_id: z.string().describe("The Node ID of the user"),
+  avatar_url: z.string().describe("The avatar URL of the user"),
+  gravatar_id: z.string().describe("The Gravatar ID of the user"),
+  url: z.string().describe("The API URL for this user"),
+  html_url: z.string().describe("The HTML URL to view this user on GitHub"),
+  followers_url: z.string().describe("The API URL for followers of this user"),
+  following_url: z.string().describe("The API URL for following of this user"),
+  gists_url: z.string().describe("The API URL for gists of this user"),
+  starred_url: z
+    .string()
+    .describe("The API URL for starred repositories of this user"),
+  subscriptions_url: z
+    .string()
+    .describe("The API URL for subscriptions of this user"),
+  organizations_url: z
+    .string()
+    .describe("The API URL for organizations of this user"),
+  repos_url: z.string().describe("The API URL for repositories of this user"),
+  events_url: z.string().describe("The API URL for events of this user"),
+  received_events_url: z
+    .string()
+    .describe("The API URL for received events of this user"),
+  type: z.string().describe("The type of this user"),
+  site_admin: z.boolean().describe("Whether this user is a site administrator"),
+  score: z.number().describe("The search result score"),
+});
+
+/**
+ * Response schema for user search results
+ */
+export const SearchUsersResponseSchema = z.object({
+  total_count: z.number().describe("Total number of matching results"),
+  incomplete_results: z
+    .boolean()
+    .describe("Whether the results are incomplete"),
+  items: z.array(SearchUserItemSchema).describe("The search results"),
+});
+
+/**
+ * Input schema for code search
+ * @see https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-code--parameters
+ */
+export const SearchCodeSchema = z.object({
+  q: z
+    .string()
+    .describe(
+      "Search query. See GitHub code search syntax: https://docs.github.com/en/search-github/searching-on-github/searching-code"
+    ),
+  sort: z
+    .enum(["", "indexed"])
+    .optional()
+    .describe("Sort field. Only 'indexed' is supported"),
```

Comment:
If only `indexed` is supported, seems like we should just remove this?

Reply:
agreed, will remove.

Reply:
done.